src/components/molecules/Visualizer/hooks.ts

```diff
@@ -105,38 +110,37 @@ export default ({
   dropRef(wrapperRef);
 
   const viewportMobileMaxWidth = 768;
-  const defaultViewport = useMemo(
-    () => ({
-      width: 1080,
-      height: 720,
-      isMobile: false,
-    }),
-    [],
-  );
-  const [viewport, setViewport] = useState<Viewport>(defaultViewport);
+  const [viewport, setViewport] = useState<VisualizerViewport>({ isMobile: false });
 
   const viewportResizeObserver = useMemo(
```

Comment:
Can you move viewportResizeObserver inside following useEffect?

Reply:
Updated.